test/language/computed-property-names/computed-property-names-prototype.js

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) Copyright 2014 the V8 project authors. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+/*---
+es6id: 12.2.5
+description: >
+    computed property names can be "prototype"
+---*/
+class C {
+  static ['prototype']() {
```

Comment:
static methods cannot be named "prototype", because the .prototype property is non-writable + non-configurable, cf. 14.5.14 step 15, 9.2.9 step 8.

js> (class C { static ['prototype']() { } })
uncaught exception: TypeError: cannot create property "prototype"